core/src/components/alert/alert.ios.scss

```diff
@@ -123,7 +123,7 @@
 }
 
 .alert-tappable {
-  height: $alert-ios-tappable-height;
+  min-height: $alert-ios-tappable-height;
```

Comment:
The ticket notes that we should set `height: auto`, but I found that created some undesired diffs on radios/checkboxes with short text. I found that `min-height` gets us to the same desired result without the additional diffs for short text.